src/main/java/seedu/address/ui/HomeCard.java

```diff
@@ -147,42 +154,17 @@ private void setUpAnimation() {
      */
     public ObservableList<Person> getAvailableTodayList() {
 
-        Day filterDay = Day.MONDAY;
         DateTimeFormatter dayFormatter = DateTimeFormatter.ofPattern("EEEE");
         String today = LocalDateTime.now().format(dayFormatter);
-        switch (today) {
-        case "Monday":
-            filterDay = Day.MONDAY;
-            break;
-        case "Tuesday":
-            filterDay = Day.TUESDAY;
-            break;
-        case "Wednesday":
-            filterDay = Day.WEDNESDAY;
-            break;
-        case "Thursday":
-            filterDay = Day.THURSDAY;
-            break;
-        case "Friday":
-            filterDay = Day.FRIDAY;
-            break;
-        case "Saturday":
-            filterDay = Day.SATURDAY;
-            break;
-        case "Sunday":
-            filterDay = Day.SUNDAY;
-            break;
-        default:
-            break;
-        }
+        Day filterDay = Day.getDay(today);
```

Comment:
You have used the `getDay` method from `Day` class, which is great. But you don't really need the next line anymore right? `final Day effectiveFilterDay = filterDay;` because filterDay itself is already effectively final.